producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/submitquery/SubmitQueryConfigThree.java

```diff
@@ -0,0 +1,123 @@
+package uk.gov.moj.sdt.producers.comx.config.submitquery;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.moj.sdt.domain.ErrorLog;
+import uk.gov.moj.sdt.domain.SubmitQueryRequest;
+
+@Configuration
+@EnableAutoConfiguration
+public class SubmitQueryConfigThree {
+
+    @Bean("uk.gov.moj.sdt.domain.api.ISubmitQueryRequestMCOLDefence3")
+    public SubmitQueryRequest submitQueryRequest() {
+        SubmitQueryRequest submitQueryRequest = new SubmitQueryRequest();
+        submitQueryRequest.setResultCount(10);
+        submitQueryRequest.setErrorLog(errorLog());
+        submitQueryRequest.setStatus("Error");
+        String response = """
+            <![CDATA[
+                              <sub:mcolDefenceDetail xmlns:sub="http://ws.sdt.moj.gov.uk/2013/sdt/SubmitQueryResponseSchema" xmlns:mquer="http://ws.sdt.moj.gov.uk/2013/mcol/QuerySchema">
```

Comment:
Can reduce the level of indentation of the contents of CDATA to make it more readable, e.g.
```
<![CDATA[
  <sub:mcolDefenceDetail xmlns:sub="http://ws.sdt.moj.gov.uk/2013/sdt/SubmitQueryResponseSchema" ...
```

A number of lines in the text block are indented with a mixture of tabs and spaces.  Is it worth changing the tabs to spaces to make the spacing consistent?

Reply:
doe